Rom.py

```diff
@@ -501,7 +522,11 @@ def patch_rom(world, player, rom):
                 else:
                     # patch door table
                     rom.write_byte(0xDBB73 + exit.addresses, exit.target)
-
+    if world.mode == 'inverted':
+        patch_shuffled_dark_sanc(world, rom, player)
+    if world.shuffle_links_house is True:
```

Comment:
Isn't world.shuffle_links_house a dictionary rather than a bool (in order to support multiworld)?

Reply:
Yes, but I've removed this entirely since the function seems unnecessary